src/Audit/UntrustedRoles.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drutiny\Plugin\Drupal7\Audit;
+
+use Drutiny\Audit;
+use Drutiny\Sandbox\Sandbox;
+use Drutiny\Annotation\Param;
+
+/**
+ * Generic module is disabled check.
```

Comment:
Not sure this is a valid description of the class?

Reply:
ha! copy/paste from your param example in the docs, forgot to update that part.